docs/fsharp/style-guide/formatting.md

```diff
@@ -646,14 +646,21 @@ if a then
 if a then ()
 ```
 
-If any of the expressions are multi-line or `if/then/else` expressions.
+If any of the expressions are multi-line, each conditional branch should be multi-line.
 
 ```fsharp
 // ✔️ OK
 if cond then
+    let e1 = something()
```

Comment:
Both of these sections were describing how to format expressions when the content was multi-line, but neither actually had a multi-line expression in them, so that's why I added these `let e1 = something()` lines.

I wasn't sure if I should update this for both of these cases, since they're essentially describing the same thing, one just includes some `elif` branches, but I aired on the side of including both for completeness' sake.